### PR TITLE
[Test Lib] Added verify starburst beta text test, fixed Intel tile, fixed pachyderm get started link and removed IBM watson studio resource.

### PR DESCRIPTION
**Pull request by risusing**

### CODE REVIEW

File: tests/Tests/600__ai_apps/680__starburst/680__starburst.robot [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+*** Test Cases ***
+Verify if the Starburst Beta text has been removed from Getting Started
+    [Documentation]
+
+    [Tags]  ODS-1158
+    Click Link  Explore
+    Wait Until Cards Are Loaded
+    Open Get Started Sidebar And Return Status    card_locator=${SB_CARDS_XP}
+    Page Should not Contain Element     xpath:${SB_BETA_DESC}
```
Comment (bdattoma) [inline tests/Tests/600__ai_apps/680__starburst/680__starburst.robot:20]: hm to be consistent with the approach used in ODS-488 to do the check based on the RHODS version, I would do the same here. So, if RHODS version is greater than 1.8.0, then starburst must NOT have the beta desc. Else, it should.

This imply a change in the name of the test case

Comment (risusing) [reply tests/Tests/600__ai_apps/680__starburst/680__starburst.robot]: Yes I have updated it. Can you review it again and tell me anything needs to be changed?

Comment (bdattoma) [reply tests/Tests/600__ai_apps/680__starburst/680__starburst.robot]: the check on beta version should work in this way:
1. if RHODS >= v1.8.0 => starburst should not have Beta text (ODS-1158)
2. fi RHODS < v1.8.0 => starburst should have Beta text (ODS-605)

(pls avoid IF/ELSE in test cases. Have a look at the [style guide ](https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/)

Comment (risusing) [reply tests/Tests/600__ai_apps/680__starburst/680__starburst.robot]: Actually I saw this kind of test cases that's why tried it. Anyways I have updated it.

Comment (bdattoma) [reply tests/Tests/600__ai_apps/680__starburst/680__starburst.robot]: thanks!

there are still Test cases to be aligned with the new style guide, that's why

File: tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot [hunk 2]
```diff
@@ -137,7 +138,12 @@ Check HTTP Status Code
     [Return]  ${response.status_code}
 
 Load Expected Data Of RHODS Explore Section
-    ${apps_dict_obj}=  Load Json File  ${APPS_DICT_PATH}
+    ${version-check}=   Is RHODS Version Greater Or Equal Than  1.8.0
+    IF  ${version-check}==True
+        ${apps_dict_obj}=  Load Json File  ${APPS_DICT_PATH_LATEST}
+    ELSE
+        ${apps_dict_obj}=  Load Json File  ${APPS_DICT_PATH}
+    END
```
Comment (lugi0) [inline tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:146]: Is the `LATEST` version of the JSON supposed to be used from 1.8 onwards? or did you mean 1.7?

File: tests/Tests/600__ai_apps/680__starburst/680__starburst.robot [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+
+*** Keywords ***
+Check Beta Description
+    ${version-check}=    Is RHODS Version Greater Or Equal Than    1.8.0
+    IF    ${version-check}==True
+        Page Should not Contain Element    xpath:${SB_BETA_DESC}
+    ELSE
+        Page Should Contain Element    xpath:${SB_BETA_DESC}
+    END
```
Comment (lugi0) [inline tests/Tests/600__ai_apps/680__starburst/680__starburst.robot:32]: Is this supposed to be 1.8 or did you mean 1.7?

File: tests/Tests/600__ai_apps/680__starburst/680__starburst.robot [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+
+*** Variables ***
+${SB_CARDS_XP}=     //article[@id="starburst"]
+${SB_BETA_DESC}=    //*[@class="pf-c-drawer__panel-main"]//div[@class="pf-c-alert pf-m-inline pf-m-info"]
+
+
+*** Test Cases ***
+Verify if the Starburst Beta text has been removed from Getting Started
+    [Tags]    ODS-1158    ODS-605
```
Comment (lugi0) [inline tests/Tests/600__ai_apps/680__starburst/680__starburst.robot:17]: This doesn't have any Tier tags and as such would never be run. Shouldn't we at least tag it with Tier2 or Sanity?

File: tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot (hunk 2 above)
Comment (bdattoma) [reply tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:146]: the intent is to use from 1.8 onwards

File: tests/Tests/600__ai_apps/680__starburst/680__starburst.robot (hunk 3 above)
Comment (bdattoma) [reply tests/Tests/600__ai_apps/680__starburst/680__starburst.robot:32]: 1.8

File: tests/Tests/600__ai_apps/680__starburst/680__starburst.robot (hunk 4 above)
Comment (bdattoma) [reply tests/Tests/600__ai_apps/680__starburst/680__starburst.robot:17]: it could be considered as Sanity

**Comment by risusing** [comment]
Jenkins job run : 
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/483/

**Comment by risusing** [comment]
Test on RHODS v1.8.0:
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/496/

**Comment by lugi0** [comment]
@risusing You will need to update this PR to include IBM Watson resources. Contact @bdattoma for more information.

**Pull request closed without merging**